ods_ci/utils/scripts/testconfig/generateTestConfigFile.py

```diff
@@ -261,6 +261,17 @@ def generate_test_config_file(
     data["OPERATOR_NAMESPACE"] = config_data["OPERATOR_NAMESPACE"]
     data["NOTEBOOKS_NAMESPACE"] = config_data["NOTEBOOKS_NAMESPACE"]
     data["OPENSHIFT_PIPELINES_CHANNEL"] = config_data["OPENSHIFT_PIPELINES_CHANNEL"]
+
+    data["CLUSTER_TYPE"] = config_data["CLUSTER_TYPE"]
+    data["TEST_ENV"] = config_data["TEST_ENV"]
+    data["INSTALL_TYPE"] = config_data["INSTALL_TYPE"] if config_data.get("INSTALL_TYPE") else "OperatorHub"
```

Comment:
we haven't had need for it so far, why this is necessary now?

Reply:
in your pr for CI, it is still passing the parameter, so maybe this change is not necessary now. Up to you, important thing is that we don't introduce new regression risk for things we won't use